task.css

```diff
@@ -59,7 +62,7 @@ body{
     
 }
 .box2{
```

Comment:
rename box-<n> class to box-order-<n> as it just taking care of order or you can define as per some meaningful order name like top-bottom or left-right instead of number sequence;